File: python/1113.py
# -*- coding: utf-8 -*-

l = [1, 2]
while l[0] != l[1]:
    l = [int(i) for i in input().split()]
    if l[0] > l[1]:
        print('Decrescente')
    elif l[1] > l[0]:
        print('Crescente')

File: c++/1113.cpp
#include <iostream>

using namespace std;

int main() {
    int x, y;

    cin >> x >> y;
    while (x != y) {
        if (x > y) {
            cout << "Decrescente\n";
        } else if (x < y) {
            cout << "Crescente\n";
        }
        cin >> x >> y;
    }

    return 0;
}